etc/stack-schema-map.yaml

```diff
@@ -9,4 +9,4 @@
 
 "7.14.0":
   beats: "master"  # TODO: 7.14.x
-  ecs: "1.10.0"
+  ecs: "master"  # TODO: master came out after 7.13.0 release
```

Comment:
shouldn't "master" be 7.15.0

Reply:
7.14 beats are not released yet

Reply:
i'm a little confused, but that's okay and not relevant here